lib/Makefile.am

```diff
@@ -137,6 +137,8 @@ install-exec-hook: qblog_script.ld
 	  sed -i -- "s/libqb.so.<digit>/$${so_ver}/" \
 	    "$(DESTDIR)$(libdir)/libqb.so-t" "$(DESTDIR)$(pkgconfigexecdir)/libqb.pc"
 	mv -f "$(DESTDIR)$(libdir)/libqb.so-t" "$(DESTDIR)$(libdir)/libqb.so"
+	rm -f "$(DESTDIR)$(pkgconfigexecdir)/libqb.pc--"
+	rm -f "$(DESTDIR)$(libdir)/libqb.so-t--"
 endif
```

Comment:
Took me some moments to realize the issue -- there's an ambiguity
across systems as to whether `--` should be considered as an argument
to `-i` option or whether this option should be considered orphan (not
accompanied with any argument as it's optional, anyway), followed with
`delimiter indicating the end of options`:
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02

Reply:
Do we have a better answer to this other than just removing the junk files? That chunk of shell code is extremely opaque (I remember commenting on that at the time.)

Reply:
Let me include the fix for that into #288 I am going to overhaul a bit
(incl. a solution for backdated OpenBSD per #299).  We need to get
rid of `sed -i` as it's not POSIX :-/

Reply:
Ah, don't get me started ... spent way too much time yesterday trying to come up with a portable function to replace sed -i and gave up -- it's not impossible, but far uglier than this simple workaround:

    sed ... "$INFILE" > "${INFILE}.$$"
    mv -- "${INFILE}.$$" "$INFILE"

It obviously only works on one input file at a time. If you need to preserve permissions, you have to do this first:

    cp -p "$INFILE" > "${INFILE}.$$"

(A portable way to preserve permissions without copying the entire file is also too ugly for words.)

Reply:
If you had suspicion what triggered this response of mine, you were right,
I came by that pacemaker commit, thanks for insights.